### PR TITLE
CHAPDEV-133 add new jobs to test gcc 4.7, 4.9, 5.4, and 6.1

### PR DESCRIPTION
**Pull request by awallace-cray**

### CODE REVIEW

**Comment by awallace-cray** [comment]
tested with gcc 4.7 and gcc 4.9, on chapcs11 and chap15

File: util/cron/test-linux64-gcc47.examples.bash [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Test default configuration on examples only, on linux64, with compiler gcc-4.7
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="linux64-gcc47.examples"
+
+gcc_setup=/data/cf/chapel/setup_gcc47.bash
+if source "${gcc_setup}"
```
Comment (ronawho) [inline util/cron/test-linux64-gcc47.examples.bash]: Is there a simple way to put all the "load gcc X and error on failure" logic into a common file so in here you would just see something like 

```sh
source $CWD/common-load-gcc.bash
load_gcc(<version>)
```

**Comment by awallace-cray** [comment]
this on hold: reviewing some test logs, I'm not sure the environment set by /data/cf/chapel/setup_gcc* actually survives the test harness.

**Comment by awallace-cray** [comment]
This should do it. Manual tests are running now.

**Comment by ronawho** [comment]
👍 thanks

**Comment by awallace-cray** [comment]
08/24/2016:

Hey Tony,

gcc 6.2 (a bugfix only release) has been installed. Can you swap out the gcc
6.1 testing for gcc 6.2

Thanks,
Elliot

**Pull request merged**